lib/Minz/Mailer.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+require LIB_PATH . '/PHPMailer/PHPMailer.php';
+require LIB_PATH . '/PHPMailer/Exception.php';
+require LIB_PATH . '/PHPMailer/SMTP.php';
+
+/**
+ * Allow to send emails.
+ *
+ * The Minz_Mailer class must be inherited by classes under app/Mailers.
+ * They work similarly to the ActionControllers in the way they have a view to
+ * which you can pass params (eg. $this->view->foo = 'bar').
+ *
+ * The view file is not determined automatically, so you have to select one
+ * with, for instance:
+ *
+ * ```
+ * $this->view->_path('user_mailer/email_need_validation.txt')
+ * ```
+ *
+ * Minz_Mailer uses the PHPMailer library under the hood. The latter requires
+ * PHP >= 5.5 to work. If you instantiate a Minz_Mailer with PHP < 5.5, a
```

Comment:
We should add a note in the main readme(s) + documentation

Reply:
I added documentation in [the other PR](https://github.com/FreshRSS/FreshRSS/pull/2481/files#diff-058bbce9aa164d0e426a3e742165070e) since this one is just technical (there is no impact for admins yet)